### PR TITLE
Adding docs for changing server config via the UI & cli

### PR DESCRIPTION
**Pull request by kdhamric**
Adding docs for changing server config via the UI & cli for 3 pages:
- demo config
- analytics config
- polling config

### CODE REVIEW

File: docs/docs/configuration/analytics.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+## Changing Analytics Settings with the CLI
+
+Or, if you prefer using the CLI, you can use this file config to disable analytics:
+
+```yaml
+type: Config
+spec:
+  analyticsEnabled: false
+  id: current
+  name: Config
```
Comment (schoren) [inline docs/docs/configuration/analytics.md:26]: this is not neccesary, it gets autopopulated 
```suggestion
```

File: docs/docs/configuration/trace-polling.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+## Changing Trace Polling Settings with the CLI
+
+Or, if you prefer using the CLI, you can use this resource definition to set the polling parameters for the Tracetest server:
+
+```yaml
+type: PollingProfile
+spec:
+  default: true
+  id: current
+  name: default
```
Comment (schoren) [inline docs/docs/configuration/trace-polling.md:51]: same for this, not neccesary
```suggestion
```

**Review by schoren — APPROVED** [review]
looks great, left 2 suggestions for fields that are not strictly requried, but feel free to merge as is

File: docs/docs/configuration/analytics.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+
+To improve the end user experience and to help the team decide where to focus resources to improve the tool, Tracetest collects analytics and telemetry information from the system.
+
+Participation in this program is optional, and you may opt-out by following the directions below if you'd prefer not to share any information.
+
+The data collected is anonymous and is not traceable to the source. You can learn more about how we treat your data by [reading our privacy statement](https://kubeshop.io/privacy).
+
+## Changing Analytics Settings from the UI
+
+In the Web UI, open settings, and select Analytics.
```
Comment (jfermi) [inline docs/docs/configuration/analytics.md:11]:
```suggestion
In the Web UI, open Settings and select Analytics:
```

File: docs/docs/configuration/analytics.md [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+
+![Analytics Settings](./img/analytics-settings.png)
+
+From this analytics settings page you can enable or disable the analytics.
```
Comment (jfermi) [inline docs/docs/configuration/analytics.md:15]:
```suggestion
From this analytics settings page, you can enable or disable the analytics.
```

**Review by xoscar — APPROVED** [review]

File: docs/docs/configuration/analytics.md [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+
+## Changing Analytics Settings with the CLI
+
+Or, if you prefer using the CLI, you can use this file config to disable analytics:
```
Comment (jfermi) [inline docs/docs/configuration/analytics.md:19]:
```suggestion
Or, if you prefer to us the CLI, you can use this file config to disable analytics:
```

File: docs/docs/configuration/demo.md [hunk 6]
```diff
@@ -0,0 +1,52 @@
+# Demo Settings
+
+Tracetest has the option to enable Test examples for our Pokeshop Demo App or the OpenTelemetry Astronomy Shop Demo. You will require an instance of those applications running alongside your Tracetest server to be able to use them. The demo settings can be adjust both from the Tracetest UI and from the CLI.
```
Comment (jfermi) [inline docs/docs/configuration/demo.md:3]:
```suggestion
Tracetest has the option to enable Test examples for our Pokeshop Demo App or the OpenTelemetry Astronomy Shop Demo. You will require an instance of those applications running alongside your Tracetest server to be able to use them. The demo settings can be adjusted both from the Tracetest UI and from the CLI. 
```

File: docs/docs/configuration/demo.md [hunk 7]
```diff
@@ -0,0 +1,52 @@
+# Demo Settings
+
+Tracetest has the option to enable Test examples for our Pokeshop Demo App or the OpenTelemetry Astronomy Shop Demo. You will require an instance of those applications running alongside your Tracetest server to be able to use them. The demo settings can be adjust both from the Tracetest UI and from the CLI. 
+
+## Changing Demo Settings from the UI
+
+In the Web UI, open settings, and select Demo.
```
Comment (jfermi) [inline docs/docs/configuration/demo.md:7]:
```suggestion
In the Web UI, open Settings and select Demo:
```

File: docs/docs/configuration/demo.md [hunk 8]
```diff
@@ -0,0 +1,52 @@
...
+
+![Demo Settings](./img/demo-settings.png)
+
+From this demo settings page you can enable the Pokeshop and/or OpenTelemetry Astronomy Shop demos. For each demo, you will need to tell Tracetest the location of the relevant demo's endpoints, adjusting the domain name / ip address and possibly the port to match your configuration.
```
Comment (jfermi) [inline docs/docs/configuration/demo.md:11]:
```suggestion
From this demo settings page, you can enable the Pokeshop and/or OpenTelemetry Astronomy Shop demos. For each demo, you will need to tell Tracetest the location of the relevant demo's endpoints, adjusting the domain name/ip address and, possibly, the port to match your configuration.
```

File: docs/docs/configuration/analytics.md [hunk 9]
```diff
@@ -0,0 +1,34 @@
+# Analytics Settings
+
+To improve the end user experience and to help the team decide where to focus resources to improve the tool, Tracetest collects analytics and telemetry information from the system.
```
Comment (jfermi) [inline docs/docs/configuration/analytics.md:3]:
```suggestion
To improve the end user experience and to help the determine where to focus team resources to improve the tool, Tracetest collects analytics and telemetry information from the system.
```

File: docs/docs/configuration/demo.md [hunk 10]
```diff
@@ -0,0 +1,52 @@
...
+
+## Changing Demo Settings with the CLI
+
+Or, if you prefer using the CLI, you can use this resource definition to enable the Pokemon app demo:
```
Comment (jfermi) [inline docs/docs/configuration/demo.md:15]:
```suggestion
Or, if you prefer using the CLI, you can use this resource definition file to enable the Pokemon app demo:
```

**Review by jfermi — COMMENTED** [review]
Some editing suggestions.

File: docs/docs/configuration/trace-polling.md [hunk 11]
```diff
@@ -0,0 +1,62 @@
+# Trace Polling Settings
+
+Tracetest currently has one algorithm for gathering the trace associated with a test which is based on periodic polling. We will be adding the ability to add multiple polling settings in the coming releases, as well as adding new polling strategies. Let us know as you have specific needs by [adding an issue](https://github.com/kubeshop/tracetest/issues/new/choose).
+
+## Periodic Polling Strategy
+
+Tracetest polls the trace data store to find and retrieve the trace that is generated by the current test run. Both the total length of time to wait to receive the trace, ie the 'Max wait time for trace', and the time to delay between each polling attempt, ie 'Retry delay', can be adjusted. The following diagram shows how the polling for the trace works:
```
Comment (jfermi) [inline docs/docs/configuration/trace-polling.md:7]:
```suggestion
Tracetest polls the trace data store to find and retrieve the trace that is generated by the current test run. Both the total length of time to wait to receive the trace, i.e., the 'Max wait time for trace' and the time to delay between each polling attempt, i.e., 'Retry delay', can be adjusted. The following diagram shows how the polling for the trace works:
```

File: docs/docs/configuration/trace-polling.md [hunk 12]
```diff
@@ -0,0 +1,62 @@
...
+    K --> O[Generating Outputs]
+    O --> P[Running Test Specs]
+    P --> Q[Finish]
+    L --> ES
+```
+  
+
+## Changing Trace Polling Settings from the UI
+
+In the Web UI, open settings, and select Trace Polling.
```
Comment (jfermi) [inline docs/docs/configuration/trace-polling.md:36]:
```suggestion
In the Tracetest Web UI, open Settings and select Trace Polling:
```

File: docs/docs/configuration/trace-polling.md [hunk 13]
```diff
@@ -0,0 +1,62 @@
...
+
+![Demo Settings](./img/trace-polling.png)
+
+From this trace polling page under settings you can adjust the speed that the trace data store is polled when gathering the trace associated with a test run.
```
Comment (jfermi) [inline docs/docs/configuration/trace-polling.md:40]:
```suggestion
From this trace polling page, under settings, you can adjust the speed with which the trace data store is polled when gathering the trace associated with a test run.
```

File: docs/docs/configuration/trace-polling.md [hunk 14]
```diff
@@ -0,0 +1,62 @@
...
+
+## Changing Trace Polling Settings with the CLI
+
+Or, if you prefer using the CLI, you can use this resource definition to set the polling parameters for the Tracetest server:
```
Comment (jfermi) [inline docs/docs/configuration/trace-polling.md:44]:
```suggestion
Or, if you prefer using the CLI, you can use this resource definition file to set the polling parameters for the Tracetest server:
```

File: docs/docs/configuration/trace-polling.md [hunk 15]
```diff
@@ -0,0 +1,62 @@
...
+  default: true
+  id: current
+  name: default
+  periodic:
+    retryDelay: 500ms
+    timeout: 50s
+  strategy: periodic
+```
+
+Proceed to run this command in the terminal, and specify the file above.
```
Comment (jfermi) [inline docs/docs/configuration/trace-polling.md:58]:
```suggestion
Proceed to run this command in the terminal and specify the file above.
```

File: docs/docs/configuration/analytics.md [hunk 16]
```diff
@@ -0,0 +1,34 @@
...
+
+```yaml
+type: Config
+spec:
+  analyticsEnabled: false
+  id: current
+  name: Config
+```
+
+Proceed to run this command in the terminal, and specify the file above.
```
Comment (jfermi) [inline docs/docs/configuration/analytics.md:27]:
```suggestion
Proceed to run this command in the terminal and specify the file above.
```

**Review by jfermi — COMMENTED** [review]
Some editing suggestions.

**Pull request merged**